[TASK] Document proper file_list.primaryActions TS settings (#542)
TYPO3-Documentation/TYPO3CMS-Reference-TSconfig: Documentation/UserTsconfig/Options.rst

    :type: string
    :Default: :typoscript:`view,metadata,translations,delete`

    Option to add more primary actions to the list view,

>     which are otherwise only accessible through the "..." menu in file list

Author: ```suggestion
    which are otherwise only accessible through the "..." menu in the file list
```

    module.

    The list of actions to be displayed can be given in the TSConfig of
    the backend user. The actions that can be set are

    *   :typoscript:`copy`
    *   :typoscript:`cut`
    *   :typoscript:`delete`
    *   :typoscript:`download`
    *   :typoscript:`edit`
    *   :typoscript:`info`
    *   :typoscript:`metadata`
    *   :typoscript:`paste`
    *   :typoscript:`rename`
    *   :typoscript:`replace`
    *   :typoscript:`translations` (always active)
    *   :typoscript:`updateOnlineMedia`
    *   :typoscript:`upload`
    *   :typoscript:`view`

    Example:

    ..  code-block:: tsconfig
        :caption: EXT:site_package/Configuration/user.tsconfig

>         # This will add "copy", "cut" and "replace" buttons to the three default

Author: ```suggestion
        # This will add "copy", "cut" and "replace" buttons in addition to the three default
```

>         # values. "translations" can be omit, as it will be added by default,

Author: ```suggestion
        # buttons. "translations" can be omitted, as it will be added by default,
```
